src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -472,52 +512,181 @@ private void CreateRegionLinks()
                 if (connections.ContainsKey(connectionKey) || connections.ContainsKey(reverseConnectionKey))
                     continue;
 
-                var pathToAdjacent = GetLeastIntersectingPath(region, adjacent);
+                var (pathToAdjacent, startVertical, endVertical) = GetLeastIntersectingPath(region, adjacent);
 
-                connections.Add(connectionKey, pathToAdjacent);
+                connections.Add(connectionKey, (pathToAdjacent, startVertical, endVertical));
             }
         }
 
         AdjustPathEndpoints();
     }
 
+    private int AdjustPriority(int size1, int priority)
```

Comment:
For understandability I'd add a general comment before this group of priority methods for what the priority means, something like:

```
// The next few methods are for calculating path priorities. They are needed because ...
```

Reply:
The change you did is slightly different in semantics, an XML comment *only* applies to the language element that it is before. So now this comment only applies to AdjustPriority. So I suggest it is reformatted to be a normal comment `//`  instead of `/// <summary>` and a blank line to be added after it and before AdjustPriority to make it clear the comment applies to multiple things and not just this one method.